apps/rehabstod/src/components/Table/TableHeaderCell.tsx

```diff
@@ -18,6 +18,7 @@ export function TableHeaderCell<T extends string>({
 }) {
   return (
     <th
+      tabIndex={0}
       onClick={() => onColumnSort(column)}
```

Comment:
We could add `onKeyDown` event handler here to make enter and space key trigger the `onColumnSort` function